ethereum/contracts/pyth/PythGovernanceInstructions.sol

```diff
@@ -43,9 +44,15 @@ contract PythGovernanceInstructions {
         address newImplementation;
     }
 
-    struct SetGovernanceDataSourcePayload {
-        PythInternalStructs.DataSource newGovernanceDataSource;
-        uint64 initialSequence;
+    struct TransferGovernanceDataSourcePayload {
+        // claimVaa is a VAA with TransferGovernanceDataSourceClaimOwnership
+        // governance instruction (header + payload) that is created from
+        // the second multisig
+        bytes claimVaa;
+    }
+
+    struct TransferGovernanceDataSourceClaimPayload {
```

Comment:
I don't like this naming, because it implicitly means that these are two separate actions where as one is included in another one.

Reply:
There are two separate actions -- one emitter first has to produce the request, and then another emitter has to approve it. The approval points to the request that it's approving. I don't think there's anything weird about that. 

I don't like the current naming because you can't tell which vaa is which. I thought the claim would be the parent VAA when i was reading this initially.

Happy to brainstorm other names with you if you really don't like request/response.

Reply:
I think claim is more semantically correct. How about "Authorize"/"Claim"?

Reply:
I like `Authorize`/`Claim`. Thanks Tom!